samples/Kconfig.mcumgr_bt_ota_dfu

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU over Bluetooth"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select MCUMGR_GRP_ZEPHYR_BASIC
+	select MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	select MCUMGR_SMP_REASSEMBLY_BT
+	select NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	select OS_MGMT_MCUMGR_PARAMS
+	imply NORDIC_QSPI_NOR
+	depends on BT_PERIPHERAL
+	depends on SOC_SERIES_NRF53X
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Enable this option to include MCUmgr in the build which will enable
+	  the Bluetooth transport and image management group, which allows
+	  for firmware over the air update support of the device.
+
+if NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+
+config BT_PERIPHERAL_PREF_MIN_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_MAX_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_LATENCY
+	default 99
+
+config BT_PERIPHERAL_PREF_TIMEOUT
+	default 400
+
+config UPDATEABLE_IMAGE_NUMBER
+	default 2
+
+config MCUMGR_SMP_BT_AUTHEN
+	default n
+
+config IMG_ERASE_PROGRESSIVELY
+	default y
+
+config MCUMGR_BUF_SIZE
+	default 2475
+
+config MCUMGR_SMP_WORKQUEUE_STACK_SIZE
+	default 4608
+
+if NORDIC_QSPI_NOR
+
+config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
+	default 4096
+
+config NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE
+	default 16
+
+endif # NORDIC_QSPI_NOR
+
+endif # NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	bool "MCUmgr OTA DFU"
+	help
+	  Enable this option to speed up the OTA DFU transfer over Bluetooth.
+	  This option extends the Bluetooth buffers to extend Bluetooth MTU
+	  size. The buffer configuration should be the same in the Network Core
+	  and Application Core image.
+
+if NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+
+config BT_L2CAP_TX_MTU
+	default 498 if !BT_CTLR
+
+config BT_BUF_ACL_TX_SIZE
+	default 251
+
+config BT_BUF_ACL_RX_SIZE
+	default 502
+
+config BT_CTLR_DATA_LENGTH_MAX
+	default 251 if BT_CTLR
```

Comment:
I don't have the code checked out at present, in which case would BT_CTLR be selected vs BT_CTLR not be selected?

Reply:
So for someone to use this, they need to set `CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` on the application core, `CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` on the application core and `CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` on the network core?

Reply:
yes, that's right